src/services/identity/SitesCacheService.ts

```diff
@@ -14,81 +18,232 @@ import type { GitHubRepositoryData, RepositoryData } from "@root/types/repoInfo"
 // Changes are polled at fixed intervals, stored in cache, and served
 // to avoid long load time from live querying.
 
+const DEFAULT_PAGE_PARAMS = {
+  per_page: GH_MAX_REPO_COUNT,
+  sort: "full_name",
+  page: 1,
+}
+
+type GitHubResponseHeaders = HeadersDefaults & {
+  link?: string
+}
+
+type Link = {
+  relid: "first" | "last" | "prev" | "next"
```

Comment:
maybe we might wanna extract this out as a type called `LinkRelation`?

Reply:
Done in https://github.com/isomerpages/isomercms-backend/pull/1239/commits/e3ad470c4b86eb8bcf1e970819e958d21d0ce4e8